build.sh

```diff
@@ -31,7 +31,19 @@ fi
 MACOSX_DEPLOYMENT_TARGET=10.8
 
 echo Running $DMD...
-$DMD -ofbin/dub -g -O -w -version=DubUseCurl -version=DubApplication -Isource $* @build-files.txt
+
+declare -ax objects # array
+while read f; do
+	echo Compiling $f ...
+	object="bin/$f.o"
+	objects+=($object)
+	# compile one file
+	$DMD -of$object -c $f -Os -w -version=DubUseCurl -version=DubApplication -Isource $*
```

Comment:
Changing `-O` to `-Os` will break compiling with dmd as it does not have that flag.